### DISCUSSION TITLE
2024-06-11(화)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#134] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by seolsis** [comment]
### 완료한 일
- 상품 삭제
### 해야할 일
- 수정사항 반영, toast pull 받아서 적용
### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- toast pr 머지
- BottomSheet에 toast 얹기

### 해야할 일
- 모달 레이아웃 작업
- 홈 페이지, 검색 페이지, 찜 페이지 작업

### 어려운 일
- 파이팅!! 💪

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
마이페이지 -> 리뷰 작성 페이지
마이페이지 -> 리뷰 상세보기 페이지

### 어려운 일

**Comment by Seoin02** [comment]
### 완료한 일
- my info 페이지 UI
### 해야할 일
- my profile 페이지 UI
- 인증 처리 API 연결 후 테스트
- 프로필 이미지 삽입 presigned URL 사용
### 어려운 일
- 인증

**Comment by q45402sk** [comment]
### 완료한 일
bottomSheet

### 해야할 일
상품상세 페이지

### 어려운 일
